components/ruby/lib/chef-licensing/license_key_fetcher/file.rb

```diff
@@ -208,9 +214,23 @@ def read_license_key_file
         # only checking for major version for file format for breaking changes
         @contents ||= YAML.load(::File.read(path))
         if major_version(@contents[:file_format_version]) == major_version(LICENSE_FILE_FORMAT_VERSION)
```

Comment:
there is one more case, where if the license file is at recent most version and our inspec is run with older version. Should we be throwing an error in this case?

Reply:
If InSpec is running with an older version of supported version it would go to the `elsif` condition, else it would raise the error.